app/controllers/employees_controller.rb

```diff
@@ -11,17 +11,23 @@ def new
 
   def create
     @employee = Employee.create employee_params
-    @employee.save
-    redirect_to employees_path
+    if @employee.save
```

Comment:
You can use AR bang method [Link doc](https://api.rubyonrails.org/classes/ActiveRecord/Persistence/ClassMethods.html)

Reply:
Thanks for the advice!We will check the bang methods.